mono/metadata/gc-internals.h

```diff
@@ -131,36 +134,39 @@ void     mono_gchandle_free_domain  (MonoDomain *domain);
 typedef void (*FinalizerThreadCallback) (gpointer user_data);
 
 void* mono_gc_alloc_pinned_obj (MonoVTable *vtable, size_t size);
-void* mono_gc_alloc_obj (MonoVTable *vtable, size_t size);
```

Comment:
Something happened line-damagy here by accident. I can fix later if required..

Reply:
Oh I think I interleaved the new functions with the old in order to pair them. Git diff doesn't work so well, perhaps.

Reply:
Er, the second change adjusted the first probably.